.travis.yml

```diff
@@ -3,8 +3,14 @@ services: docker
 script: docker build . -t cs50/cli
 before_deploy: docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 deploy:
-  provider: script
+- provider: script
   script: docker push cs50/cli
+  on:
+    branch: master
+- provider: script
+  script: docker tag cs50/cli cs50/cli:ubuntu; docker push cs50/cli:ubuntu
```

Comment:
These shouldn't be the same!

Reply:
The build step is the same for all branches. It builds the image as cs50/cli. This just ensures the image is tagged ubuntu before pushing.

Reply:
Just to be sure, this should definitely be `docker tag cs50/cli cs50/cli:ubuntu`, even though former typically implies `cs50/cli:latest`?

Reply:
Yup. See https://github.com/cs50/cli/pull/20/files#diff-354f30a63fb0907d4ad57269548329e3R3!